STEP_Project/entitySelect.js

```diff
@@ -15,12 +17,15 @@ function identifyEntityClass(button)
 
 (function()
 {
-    let OUButton = document.getElementById('OU');
-    let groupButton = document.getElementById('group');
-    let userButton = document.getElementById('user');
+    if (testFlag != 'testing')
+    {
+        let OUButton = document.getElementById('OU');
```

Comment:
ouButton

Reply:
We've tried to maintain consistent naming in the project, and we've called it OU in other places/in the code history so it would be odd to change this